### PR TITLE
feat(jobs/*): Adds cloud provider env var to e2e jobs

### PR DESCRIPTION
**Pull request by jchauncey**
This will be defaulted to google for now

### CODE REVIEW

**Comment by arschles** [comment]
@jchauncey LGTM. this seems to require https://github.com/deis/k8s-claimer/pull/104 to be merged & deployed, right?

File: common.groovy [hunk 1]
```diff
@@ -34,6 +34,7 @@ defaults = [
     storageAccountKeyID: 'c9cad0b0-53dd-4d38-a832-c4b29aeaf49b',
   ],
   statusesToNotify: ['SUCCESS', 'FAILURE'],
+  "e2eRunner": [ provider: 'google', ],
```
Comment (sgoings) [inline common.groovy]: We should default to azure I think.

**Comment by arschles** [comment]
Still LGTM after your recent changes @jchauncey

File: common.groovy (hunk 1 above)
Comment (jchauncey) [reply common.groovy]: updated

**Comment by jchauncey** [comment]
reran this job and it succeeded so im merging it - https://ci.deis.io/job/jenkins-jobs-pr/95/console

**Pull request merged**